ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -92,7 +92,7 @@ function cleanup_prior() {
 
     # Delete any vpc older than 24 hrs
     echo "Cleaning up VPCs"
-    for VPC in $(ibmcloud is vpcs --resource-group-name "${RESOURCE_GROUP}" 2>&1 | grep "${RESOURCE_GROUP}" | grep -v Listing | grep -i ci-op | awk '{print $1}')
+    for VPC in $(ibmcloud is vpcs --resource-group-name "${RESOURCE_GROUP}" 2>&1 | grep "${RESOURCE_GROUP}" | grep -v Listing | grep -i x-px | awk '{print $1}')
```

Comment:
`ibmcloud is vpcs --resource-group-name "${RESOURCE_GROUP}" 2>&1 | grep "${RESOURCE_GROUP}" | grep -v Listing | grep -i x-px | awk '{print $1}'` this will work, however, for the wrong reasons - it skips the `ci-op` entries.
As it won't find ci-op vpc entries, such as `ci-op-kznz5r7i-dfd59-q8c9c-vpc` the vpc will not be deleted.